cli/tests/integration_tests.rs

```diff
@@ -230,7 +230,16 @@ fn fmt_stdin_error() {
 }
 
 // Warning: this test requires internet access.
-#[test]
+// Only run for GitHub release targets
+#[cfg_attr(
+  any(
+    all(target_os = "windows", target_arch = "x86_64"),
+    all(target_os = "macos", target_arch = "x86_64"),
+    all(target_os = "linux", target_arch = "x86_64"),
+  ),
+  test
+)]
```

Comment:
I think this is something that's better added when we are trying to add a aarch build. It's not necessary for fixing #5369

Reply:
Shall I drop the commit from this branch?

I guess this is more related to #4862

Reply:
Dropped commit